package.json

```diff
@@ -15,10 +15,13 @@
   },
   "scripts": {
     "test": "jest",
-    "lint": "prettier-check src/*.js",
-    "format": "prettier --write src/*.js",
+    "lint": "prettier-check {demo,src}/*.{js,css}",
+    "format": "prettier --write {demo,src}/*.{js,css}",
     "watch": "watchify src/main.js --verbose --standalone envSpec -o dist/envspec.js",
-    "start": "browser-sync start --server demo --server dist --watch --files 'dist/*' 'demo/*'  --port $PORT"
+    "start": "browser-sync start --server demo --server dist --watch --files 'dist/*' 'demo/*'  --port $PORT",
+    "doc": "jsdoc src/main.js -d docs"
   },
-  "dependencies": {}
+  "dependencies": {
+    "jsdoc": "^3.5.5"
```

Comment:
Move `jsdoc` in `devDependencies` please. Explaining; env-spec-js does not depend on jsdoc itself to operate, it is a tool used by the developers of env-spec-js (us).